mu/locale/es/LC_MESSAGES/mu.po

```diff
@@ -5,756 +5,1091 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"POT-Creation-Date: 2017-09-13 17:50+0100\n"
-"PO-Revision-Date: 2017-09-13 18:40+0100\n"
+"POT-Creation-Date: 2018-07-10 09:08+0100\n"
+"PO-Revision-Date: 2018-07-10 23:16+0100\n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: pygettext.py 1.5\n"
-"X-Generator: Poedit 2.0.1\n"
+"X-Generator: Poedit 2.0.8\n"
 "Last-Translator: \n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"Language: es_ES\n"
+"Language: es\n"
```

Comment:
I've used Poedit, so not sure if this line was edited by that application or my run of pygettext. Should I revert it?

Reply:
I don't think it makes much difference, since Mu only looks at the language part of this string (rather than the whole language_LOCALE bit). If you see what I mean.

Reply:
Okay, I'll leave it as it is then, thanks.